etc/conf/mtTelemetry.yaml

```diff
@@ -0,0 +1,325 @@
+# Configuration file for the ComCam
+
+# The name of the HeaderService
+hs_name: MTHeaderService
+instrument: LSSTCam
+section: LSSTCam
+
+# Default logging level
+loglevel: 'INFO'
+
+# Path to the log location and other log definitions
+logfile: 'www/MTHeaderService.log'
+log_format: '[%(asctime)s.%(msecs)03d][%(levelname)s][%(name)s][%(funcName)s] %(message)s'
+log_format_date: '%Y-%m-%d %H:%M:%S'
+
+# path to the web server logfile
+weblogfile: 'www/webserver.log'
+
+# port_number for webserver
+port_number: 8000
+
+# The URL format to be used for lfa_mode=http
+url_format: "http://{ip_address}:{port_number}/{filename_HDR}"
+
+# While we support both s3 and http LFA we need to define the mode [s3 or http]
+lfa_mode: http
+
+# The s3 instance (i.e.: nts, tuc, ls, cp)
+# s3instance: nts
+
+# Default write_mode (fits or string)
+write_mode: yaml
+
+# The event to get the timeout
+timeout_event:
+    device: MTCamera
+    topic: startIntegration
+    value: timeout
+
+# Timeout beyond the requested exposure time to wait for an end of Telemetry event (in seconds)
+timeout_exptime: 30
+
+# The file naming event source and definitions
+filepath: 'www/DMHS_filerepo'
+format_HDR: "{}.yaml"
+format_FITS: "{}.fits"
+imageName_event:
+    device: MTCamera
+    topic: startIntegration
+    value: imageName
+
+# The Events that we use to define where we collect
+start_collection_event:
+    device: MTCamera
+    topic: startIntegration
+
+end_collection_event:
+    device: MTCamera
+    topic: endOfImageTelemetry
+
+## The event for the image readout paramters
+#imageParam_event:
+#    device: MTCamera
+#    topic: imageReadoutParameters
+#    array_keys: ccdLocation
+
+# Telemetry values for images meta-data
+# https://confluence.lsstcorp.org/display/SYSENG/ComCam+Header+Information+Topic+Mapping
```

Comment:
The link is pointing to the wrong Confluence page.

Reply:
Fixed